app/app.js

```diff
@@ -26,18 +26,25 @@ app.use(methodOverride());
 // Connect to mongodb
 mongoose.connect(database);
 
-app.use(express.static(__dirname + '/app/client'));
+app.use(express.static(__dirname + '/client'));
+
+// Enable CORS =================================================================
```

Comment:
gggggg good mans i hated trying to debug and figuring out this was the issue LOL